src/Post/postsService.js

```diff
@@ -1,25 +1,33 @@
-import Post from "./Post.js"
+import db from '../../db/index.js';
+
+const Post = db.post;
 
 class PostsService {
-    async create(post) {
-        const createdPost = await Post.create(post)
-        return createdPost
+    getAll() {
+        return Post.findAll();
+    }
+    create(post) {
+        return Post.create(post);
     }
-    async delete(id) {
+    delete(id) {
         if (!id) {
-            throw new Error("post didn't find")
+            throw new Error('post did not find');
```

Comment:
"Id is not provided" error should be here

Reply:
fixed